src/pred_set/src/pred_setScript.sml

```diff
@@ -4657,6 +4657,13 @@ val COMMUTING_ITSET_RECURSES = store_thm(
     ASM_SIMP_TAC bool_ss [COMMUTING_ITSET_INSERT, delete_non_element]
   ]);
 
+(* Corollary *)
+Theorem ITSET_SING:
```

Comment:
This should be `[simp]`

Reply:
Done.
Raises the question though: Should `ITSET_EMPTY` be a `[simp]`?

Reply:
Yes!